api/src/main/java/eu/darkbot/api/managers/StatsAPI.java

```diff
@@ -80,4 +81,9 @@ public interface StatsAPI extends API.Singleton {
      * @return earned amount of honor while bot was working
      */
     double getEarnedHonor();
+
+    class StatsResetEvent implements Event
+    {
+        public StatsResetEvent() {};
+    }
```

Comment:
```suggestion
    class StatsResetEvent implements Event {
        public StatsResetEvent() {
        }
    }
```
try to keep format consistent thru the codebase

Reply:
Yeah.. bad habits of mine to use c#(main language) formats in java.